tests/e2e/chain/config.go

```diff
@@ -0,0 +1,335 @@
+package chain
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/server"
+	srvconfig "github.com/cosmos/cosmos-sdk/server/config"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/spf13/viper"
+	tmconfig "github.com/tendermint/tendermint/config"
+	tmjson "github.com/tendermint/tendermint/libs/json"
+	"github.com/tharsis/ethermint/server/config"
+
+	"github.com/tharsis/evmos/v4/tests/e2e/util"
+)
+
+type ValidatorConfig struct {
+	Pruning            string // default, nothing, everything, or custom
+	PruningKeepRecent  string // keep all of the last N states (only used with custom pruning)
+	PruningInterval    string // delete old states from every Nth block (only used with custom pruning)
+	SnapshotInterval   uint64 // statesync snapshot every Nth block (0 to disable)
+	SnapshotKeepRecent uint32 // number of recent snapshots to keep and serve (0 to keep all)
+}
+
+const (
+	// common
+	CoinDenom     = "ucoin"
+	StakeDenom    = "stake"
+	IbcDenom      = "ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518"
```

Comment:
let's change these constants to Evmos (`aevmos` + `erc20/0x...`) and Osmosis (`ibc/...`) for accuracy